src/lib/holocene/radio-input/radio-input.svelte

```diff
@@ -29,6 +30,11 @@
   const { name, group } = ctx;
 </script>
 
+{#if description}
+  <p class="description" aria-describedby="{id}-label">
+    {description}
+  </p>
+{/if}
```

Comment:
I think we want this to have the unique `id`, and then put `aria-describedby` on the `<input />`

```
<span id="{id}-description">{description}</span>

<input aria-describedby="{id}-description" />
```